assembl/static2/js/app/pages/idea.jsx

```diff
@@ -54,8 +54,35 @@ const noRowsRenderer = () => {
 };
 
 class Idea extends React.Component {
+  topPostsToLoad = (topPosts, edges) => {
+    const { hash } = window.location;
+    if (hash !== '') {
+      const id = hash.replace('#', '');
+      const post = edges.filter((edge) => {
+        return edge.node.id === id;
+      });
+      if (post.length === 0) {
+        return null;
+      }
+      let topPostId = post[0].node.parentId;
```

Comment:
it's more `parentPostId` here than `topPostId`, isn't it? 

Reply:
from what I understand, you are really searching for the top post for this post. So it wont work for a post that is at level 3 or more ? you have to recursively search for the first post in the ancestors of `post` that have no `parentId`

Reply:
Ah you're right! I need to recursively do it.

Reply:
So yeah, it's really topPostId I want here.